tianshou/env/gym_wrappers.py

```diff
@@ -0,0 +1,32 @@
+import gym
+import numpy as np
+
+
+class DiscreteToContinuous(gym.ActionWrapper):
```

Comment:
I think it is better to name as `ContinuousToDiscrete`. Because we use it as:
```python
env = ContinuousToDiscrete(env)
```

This indicates an env transformation from continuous action space to discrete one.